src/style.rs

```diff
@@ -232,7 +238,7 @@ impl ProgressStyle {
                 } => {
                     buf.clear();
                     if let Some(formatter) = self.format_map.get(key.as_str()) {
-                        buf.push_str(&formatter(state));
+                        buf.push_str(&formatter(state).replace('\t', &" ".repeat(self.tab_width)));
```

Comment:
In #420 this is changed to pass in a `Formatter` directly. How do we change tab expansion in that case?